### ISSUE TITLE
Account keeps disappearing 

### ISSUE THREAD

**Issue by andyxheli** [nextcloud/talk-android#2533]
### Steps to reproduce

Add account for my nextcloud instance 
Add account from Nextcloud cloud server 

### Expected behaviour

Show not happen

### Actual behaviour

After a while the second account no longer shows up I added it again and it dose show. I cleared data and cache and it works for a while

### Device brand and model

SM-G996U1 

### Android version

12

### Nextcloud Talk app version

15.0.1

### Nextcloud server version

24.0.6

### Talk version

14.0.5

### Custom TURN server configured

Yes

### Custom STUN server configured

Yes

**Comment by timkrueger** [comment]
I work with three accounts in parallel and can't recognize issues like that. Can you enable my test user on your instance again?

**Comment by juergenscholz** [comment]
Same for me: have added 2 Nextcloud accounts in the past and after an update of the app only 1 account was left. Tried to add the missing account again but only 1 account can be seen after adding it.

**Comment by mahibi** [comment]
sounds like a duplicate of https://github.com/nextcloud/talk-android/issues/2446
we are aware of it and treat it with high priority. however until now it was not reproducible for me..

**Comment by mahibi** [comment]
@andyxheli 
can you try out the apk from https://github.com/nextcloud/talk-android/pull/2581#issuecomment-1320064606 and let me know if it's fixed?

**Comment by andyxheli** [comment]
> @andyxheli 
> can you try out the apk from https://github.com/nextcloud/talk-android/pull/2581#issuecomment-1320064606 and let me know if it's fixed?

Yes! I sure can ill give it a try and monitor for a few days and let you know 😉
